test/__snapshots__/test.js.snap

```diff
@@ -4393,6 +4393,191 @@ const Impl = require(\\"../implementations/UnforgeableMap.js\\");
 "
 `;
 
+exports[`Variadic.idl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const convertURL = require(\\"./URL.js\\").convert;
+const impl = utils.implSymbol;
+
+function Variadic() {
+  throw new TypeError(\\"Illegal constructor\\");
+}
+
+Object.defineProperty(Variadic, \\"prototype\\", {
+  value: Variadic.prototype,
+  writable: false,
+  enumerable: false,
+  configurable: false
+});
+
+Variadic.prototype.simple1 = function simple1() {
+  if (!this || !module.exports.is(this)) {
+    throw new TypeError(\\"Illegal invocation\\");
+  }
+
+  const args = [];
+  for (let i = 0; i < arguments.length; ++i) {
+    args[i] = arguments[i];
+  }
+  for (let i = 0; i < arguments.length; ++i) {
+    args[i] = conversions[\\"DOMString\\"](args[i], {
+      context: \\"Failed to execute 'simple1' on 'Variadic': parameter \\" + (i + 1)
+    });
+  }
+  return this[impl].simple1(...args);
+};
+
+Variadic.prototype.simple2 = function simple2(first) {
+  if (!this || !module.exports.is(this)) {
+    throw new TypeError(\\"Illegal invocation\\");
+  }
+
+  if (arguments.length < 1) {
+    throw new TypeError(
+      \\"Failed to execute 'simple2' on 'Variadic': 1 \\" + \\"argument required, but only \\" + arguments.length + \\" present.\\"
+    );
+  }
+
+  const args = [];
+  for (let i = 0; i < arguments.length; ++i) {
+    args[i] = arguments[i];
+  }
+
+  args[0] = conversions[\\"DOMString\\"](args[0], { context: \\"Failed to execute 'simple2' on 'Variadic': parameter 1\\" });
+  for (let i = 1; i < arguments.length; ++i) {
+    args[i] = convertURL(args[i], { context: \\"Failed to execute 'simple2' on 'Variadic': parameter \\" + (i + 1) });
+  }
+  return this[impl].simple2(...args);
+};
+
+Variadic.prototype.overloaded1 = function overloaded1() {
+  if (!this || !module.exports.is(this)) {
+    throw new TypeError(\\"Illegal invocation\\");
+  }
+
+  const args = [];
+  for (let i = 0; i < arguments.length; ++i) {
+    args[i] = arguments[i];
+  }
+
+  return this[impl].overloaded1(...args);
+};
+
+Variadic.prototype.overloaded2 = function overloaded2(first) {
+  if (!this || !module.exports.is(this)) {
+    throw new TypeError(\\"Illegal invocation\\");
+  }
+
+  if (arguments.length < 1) {
+    throw new TypeError(
+      \\"Failed to execute 'overloaded2' on 'Variadic': 1 \\" +
+        \\"argument required, but only \\" +
+        arguments.length +
+        \\" present.\\"
+    );
+  }
+
+  const args = [];
+  for (let i = 0; i < arguments.length; ++i) {
+    args[i] = arguments[i];
+  }
+
+  if (args[1] !== undefined) {
```

Comment:
Why not convert past the argument at index 1 like the other examples do?

Reply:
This operation is overloaded. As stated in https://github.com/TimothyGu/webidl2js/blob/6b8788120218090738d815104eaa5db584799f7a/lib/parameters.js#L80-L84 I don't want to support overloaded operations with variadic arguments just yet, especially since jsdom doesn't require it at this moment.